lib/standard/stream.nit

```diff
@@ -220,6 +220,64 @@ abstract class Reader
 	# Is there something to read.
 	# This function returns 'false' if there is something to read.
 	fun eof: Bool is abstract
+
+	# Read the next sequence of non whitespace characters.
+	#
+	# Leading whitespace characters are skipped.
+	# The first whitespace character that follows the result is consumed.
+	#
+	# An empty string is returned if the end of the file or an error is encounter.
```

Comment:
Since "" is not a valid result it is used to indicate the eof. Some other methods returns "", like read_line, in case of eof or errors.
What is the opinion of others?